src/client/components/back-button/back-button.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+  <header class="back-button__container">
+    <md-button class="md-raised" @click="$router.go(-1)">
```

Comment:
Personally I prefer the non-raised version because the space around the arrow is a bit off. Your thoughts?

Reply:
changed to default, but adjust styling so the button is left aligned to rest of content